server/src/main/java/org/opensearch/action/support/RetryableAction.java

```diff
@@ -36,13 +36,15 @@
 import org.apache.logging.log4j.message.ParameterizedMessage;
 import org.opensearch.action.ActionListener;
 import org.opensearch.action.ActionRunnable;
+import org.opensearch.action.bulk.BackoffPolicy;
```

Comment:
we might have to refactor the BackoffPolicy to different package as it in bulk action hierarchy

Reply:
Yes it should be out of bulk hierarchy. Are you suggesting to refactor along with this change, as it will require change at multiple places.

I can create new `BackoffPolicy` class out of bulk hierarchy and add our `exponentialEqualJitterBackoff` in it and we can add TODO for moving other policy to here as well. 

Thoughts?

Reply:
yes we can do that, create a new BackPolicy class in this PR and create a new github issue to migrate others as well to this new class.

Reply:
my bad, @dhwanilpatel to keep history clean, Lets keep the equal jitter policy in the BackoffPolicy class itself and add the new policy also there. Create new github issue to move the file itself to common package, that would be better.

Reply:
btw there is an ExponentialBackoff policy already in BackoffPolicy class, so there will be conflict when you merge both the classes. You can have another iterator for exponential backoff by 2 in that class.